static/css/frame.scss

```diff
@@ -535,8 +533,9 @@ body {
   box-shadow: $large-box-shadow;
   display: flex;
   justify-content: center;
-  margin: 20px auto;
   z-index: 9999;
+  position: relative;
```

Comment:
Adding position relative here is hiding top of really long shots behind editor-header, making that area inaccessible for editing (highlight, crop and pen), recommend to remove canvas relative positioning

Reply:
We can resolve this by attaching `pointer-events: none` to the `.editor-header` class

Reply:
and attaching pointer events to the little controls